tests/test_databases.py

```diff
@@ -17,23 +16,6 @@
 DATABASE_URLS = [url.strip() for url in os.environ["TEST_DATABASE_URLS"].split(",")]
 
 
-def mysql_versions(wrapped_func):
-    """
-    Decorator used to handle multiple versions of Python for mysql drivers
-    """
-
-    @functools.wraps(wrapped_func)
-    def check(*args, **kwargs):  # pragma: no cover
-        url = DatabaseURL(kwargs["database_url"])
-        if url.scheme in ["mysql", "mysql+aiomysql"] and sys.version_info >= (3, 10):
-            pytest.skip("aiomysql supports python 3.9 and lower")
```

Comment:
The description of this PR only mentions Python 3.6, but this conditional also exists there.

Reply:
Yeah aiomysql now supports Py3.10. I'll add it there.

Reply:
Done.